docs/index.rst

```diff
@@ -50,7 +50,8 @@
 
    tutorials/pwm_control_flight_controller
    tutorials/hobby_calibration
-   tutorials/dronecan_flight_controller
```

Comment:
I think you broke this link. That file doesn't exist anymore. Make sure you're pulled up to date.